version.txt

```diff
@@ -1 +1,2 @@
-0.86.0-SNAPSHOT
+0.85.2-SNAPSHOT
```

Comment:
No, the release plugin now takes care of the version file. We don't need to touch it anymore :)